src/main/webapp/WEB-INF/jsp/useractivation.jsp

```diff
@@ -56,8 +56,8 @@
                 <input class="form-control ${hasPassword2Error ? "form-control-error" : ""}" type="password" id="password2" name="password2" value="${fn:escapeXml(password2)}" autocomplete="off">
             </div>
 
-            <input class="button" type="submit" value="<spring:message code="public.common.button.continue.text"/>" id="activate">
-
+            <input class="button" type="submit" value="<spring:message code="public.common.button.continue.text"/>" id="activate"
+                onclick="setTimeout(function () {document.getElementById('activate').disabled = true;document.getElementById('activate').style.opacity='0.5';}, 0);">
```

Comment:
I think it's best to put the snippet it in a js file and reference the method from here, for code clarity.

Reply:
Would have done that, but we don't have any existing separate js files, and I'm not sure this is large enough to be worth it. I think I'll create a new PR to follow up on the 409 issues generally, and perhaps we can implement a better solution then (maybe a "please wait" popup, or something smarter)